cypress/e2e/awx/resources/jobTemplateSurveys.cy.ts

```diff
@@ -3,43 +3,59 @@ import { JobTemplate } from '../../../../frontend/awx/interfaces/JobTemplate';
 import { Organization } from '../../../../frontend/awx/interfaces/Organization';
 import { Project } from '../../../../frontend/awx/interfaces/Project';
 import { ReusableTemplateSurveyTestSuite } from './sharedTemplateSurvey';
+import { randomE2Ename } from '../../../support/utils';
 
 describe('Job Templates Surveys', function () {
   let inventory: Inventory;
   let jobTemplate: JobTemplate;
+  let organization: Organization;
+  let project: Project;
   let reusableTemplateSurveyTestSuite: ReusableTemplateSurveyTestSuite;
 
-  // FLAKY_06_13_2024
-  describe.skip('JT Surveys: Create, Edit and Delete', function () {
+  describe('JT Surveys: Create, Edit and Delete', function () {
     const question = {
       question_name: "Who's that?",
       question_description: 'The person behind this.',
       variable: 'who_is_that',
       default: 'John Doe',
       type: 'text',
-      label: 'Text',
+      max: 1024,
+      min: 0,
+      new_question: true,
+      required: true,
+      choices: [],
     };
 
     before(function () {
-      cy.createAwxInventory({ organization: (this.globalAwxOrganization as Organization).id }).then(
-        (inv) => {
-          inventory = inv;
-
-          cy.createAwxJobTemplate({
-            organization: (this.globalAwxOrganization as Organization).id,
-            project: (this.globalProject as Project).id,
-            inventory: inventory.id,
-          }).then((jT) => {
-            jobTemplate = jT;
-            reusableTemplateSurveyTestSuite = new ReusableTemplateSurveyTestSuite(jobTemplate);
-
-            cy.visit(`/templates/job-template/${jobTemplate.id}/survey`);
-          });
-        }
-      );
+      cy.createAwxOrganization(randomE2Ename()).then((org) => {
+        organization = org;
+        cy.createAwxProject({ organization: organization.id }).then((proj) => {
+          project = proj;
+        });
+      });
+    });
+
+    after(() => {
+      cy.deleteAwxOrganization(organization, { failOnStatusCode: false });
+      cy.deleteAwxProject(project, { failOnStatusCode: false });
```

Comment:
Do weed to delete the project first? Since there is an Organization associated with that one?